playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire.hosting.azure.bicep.openai.bicep

```diff
@@ -73,3 +70,6 @@ resource cognitiveServiceContributorRoleAssignment 'Microsoft.Authorization/role
 }
 
 output connectionString string = 'Endpoint=${account.properties.endpoint}'
+
+
+output connectionString string = 'Endpoint=${account.properties.endpoint}'
```

Comment:
Why is this duplicated now?

Reply:
Snap, mean automerger ... It got generated with the tool, not sure why. When looking at the diff locally I thought it was just adding a new line. Will fix it.